pkg/vc/framework.go

```diff
@@ -1,20 +1,24 @@
 package vc
 
 import (
-	"net/http"
-
 	"github.com/hyperledger/aries-framework-go/component/storageutil/mem"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/ld"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdr"
 	"github.com/hyperledger/aries-framework-go/pkg/framework/context"
 	ldstore "github.com/hyperledger/aries-framework-go/pkg/store/ld"
 	jsonld "github.com/piprate/json-gold/ld"
+	"net/http"
 )
 
 type Framework struct {
-	loader *ld.DocumentLoader
+	loader   *ld.DocumentLoader
+	resolver *verifiable.VDRKeyResolver
 }
 
-func NewFramework() (*Framework, error) {
+type FrameworkOption func(opts *Framework) error
+
+func NewFramework(vdr vdr.Registry) (*Framework, error) {
```

Comment:
The framework have verifiable data registry(VDR).
The framework will find public key paired with the key used for proof of VC/VP.